src/box/box.cc

```diff
@@ -1648,6 +1648,22 @@ box_check_wal_queue_max_size(void)
 	return size;
 }
 
+/** Check replication_synchro_queue_max_size option validity. */
+static int64_t
+box_check_replication_synchro_queue_max_size(void)
+{
+	int64_t size = cfg_geti64("replication_synchro_queue_max_size");
+	if (size < 0) {
+		diag_set(ClientError, ER_CFG,
+			 "replication_synchro_queue_max_size",
+			 "replication_synchro_queue_max_size must be >= 0");
+	}
+	/* Unlimited. */
+	if (size == 0)
+		size = INT64_MAX;
+	return size;
+}
+
 static double
```

Comment:
Let's update the commit msg of `limbo: introduce limits on synchro queue`. `queue_size` doesn't exist anymore, it's just `size`. Pointer to `journal_entry` was dropped, now it's just `approx_len` instead.

> Actually, the limit is added to an abstract `txn_limbo`, but currently
only works for the global `txn_limbo` used by the tarantool node.

I didn't get this one.

Reply:
Fixed.